packages/extension-api/src/extension-api.d.ts

```diff
@@ -959,6 +991,14 @@ declare module '@podman-desktop/api' {
      */
     export function showErrorMessage(message: string, ...items: string[]): Promise<string | undefined>;
 
+    /**
+     * Shows a message box.
+     *
+     * @param options The properties and options controlling the behavior of the message box.
+     * @return A promise that resolves to the selection index or `undefined` when being dismissed.
+     */
+    export function showMessageBox(options: MessageBoxOptions): Promise<MessageBoxReturnValue>;
+
```

Comment:
```suggestion

```
not needed

Reply:
See previous comment. I could split the new API to another PR and solely replace the existing dialog, but it didn't seen like a lot more code and shows why some of the choices were made (like calling it a message box).

Reply:
we should get rid of it

Reply:
Out of date, API removed with new commit.